File: src/main/java/kz/smarthealth/patientservice/PatientServiceApplication.java
package kz.smarthealth.patientservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@SpringBootApplication
@EnableMongoRepositories
public class PatientServiceApplication {

	public static void main(String[] args) {
		SpringApplication.run(PatientServiceApplication.class, args);
	}

}

File: src/main/kotlin/kz/samat/patientservice/PatientServiceApplication.kt
package kz.samat.patientservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@SpringBootApplication
@EnableEurekaClient
class PatientServiceApplication

fun main(args: Array<String>) {
    runApplication<PatientServiceApplication>(*args)
}
